pytorch_lightning/trainer/trainer.py

```diff
@@ -118,7 +117,6 @@
 
 class Trainer(
     TrainerCallbackHookMixin,
-    TrainerOptimizersMixin,
```

Comment:
this is really exciting to see the mixins being whittled down like this. my hope is that v1.6, all of the mixins are marked as deprecated, and the vestiges of the OG trainer design will eventually be retired (composition over inheritance!)